core/server/master/src/main/java/alluxio/underfs/MasterUfsManager.java

```diff
@@ -114,16 +116,36 @@ public synchronized void setUfsMode(Supplier<JournalContext> journalContext, All
   }
 
   @Override
-  public Iterator<JournalEntry> getJournalEntryIterator() {
-    return mState.getJournalEntryIterator();
+  public boolean processJournalEntry(JournalEntry entry) {
+    return mState.processJournalEntry(entry);
+  }
+
+  @Override
+  public void resetState() {
+    mState.resetState();
+  }
+
+  @Override
+  public CheckpointName getCheckpointName() {
+    return mState.getCheckpointName();
+  }
+
+  @Override
+  public void writeToCheckpoint(OutputStream output) throws IOException, InterruptedException {
+    mState.writeToCheckpoint(output);
```

Comment:
Should we introduce another interface that does the delegation to `mState`? I think we do this a few times.

Reply:
good idea, added